src/publisher/Publisher.ts

```diff
@@ -178,12 +161,28 @@ export class Publisher extends OTEventEmitter<{
     }
   }
 
-  destroy(): void {
+  destroy(): this {
     const call = getOrCreateCallObject();
 
-    call.leave().catch((err) => {
-      console.error(err);
+    removeParticipantMedia(call.participants().local.session_id);
```

Comment:
Do we need to do that local truthiness check here or are we guaranteed to always have local in this case?

Reply:
It won't _hurt_, it's _unlikely_ this will be an issue because calling this function will only do anything if they've already joined a meeting, but it's worth doing. I'll update it now